### ISSUE TITLE
For the docs, a CachedValueNotFound error probably means your cache isn't working

### ISSUE THREAD

**Issue by phoebebright** [EliotBerriot/django-dynamic-preferences#29]
I had this error in dev using dummy cache but once I switched to using a database cache it worked fine.

**Comment by EliotBerriot** [comment]
Yes, there are still a lot of improvements to do for the documentation, especially for the cache part. Thanks for reporting this, I'll think of a way to make the dummy cache behave normally and not raising this error.

**Comment by phoebebright** [comment]
No problem.  Do you want me to post any more comments here or send you an email?

**Comment by EliotBerriot** [comment]
Please feel free to post your comments here :)

**Comment by phoebebright** [comment]
Downing tools for Christmas but this library looks like just what I need so will come back to it later.  Thanks for your help.

**Comment by EliotBerriot** [comment]
No worry, I'd be glad to implement missing features and commit bug fixes if you need it. Also, it's totally okay to open new issues and submit Pull Requests. Merry christmas !

**Comment by EliotBerriot** [comment]
Since there is now a way to disable preference caching via settings, the documentation should probably mention that using dummycache is possible if you also set `DYNAMIC_PREFERENCES['ENABLE_CACHE'] = False`.
